src/libmscore/score.cpp

```diff
@@ -3418,7 +3418,7 @@ void Score::selectRange(Element* e, int staffIdx)
                 Segment* s1 = tick2segmentMM(t1, true, SegmentType::ChordRest);
                 Segment* s2 = tick2segmentMM(t2, true, SegmentType::ChordRest);
                 if (s2) {
-                    s2 = s2->next1MM(SegmentType::ChordRest);
+                    s2 = s2->next1MM(SegmentType::ChordRest | SegmentType::EndBarLine);
```

Comment:
Now you've changed the commit message to that of the 2nd fix, but actually kept the 1st fix ;-)